internal/orchestrationapi/orchestrationapi.go

```diff
@@ -416,15 +416,15 @@ func (orcheEngine orcheImpl) gatherDevicesResource(candidates []dbhelper.Executi
 
 			if err != nil {
 				log.Println("[orchestrationapi] cannot getting msgs from :", cand.Endpoint[0], "cause by", err.Error())
-				resources <- deviceInfo{endpoint: cand.Endpoint[0], resource: resource, id: cand.Id, execType: cand.ExecType}
+				resources <- deviceInfo{endpoint: cand.Endpoint[0], resource: resource, id: cand.ID, execType: cand.ExecType}
 				return
 			}
 			log.Printf("[orchestrationapi] deviceResource")
-			log.Printf("candidate Id       : %v", cand.Id)
+			log.Printf("candidate Id       : %v", cand.ID)
```

Comment:
The variable ID comes from ExecutionCandidate structure from helper.go file. This file generated the following lint warning 
**internal/db/helper/helper.go:37:2: struct field Id should be ID**
To remove this warning, the variable was changed to ID from Id. Hence the change in variable. Log message it can be reverted if required :)